templates/artist/includes/artist_detail_overview.html

```diff
@@ -77,8 +77,8 @@
             <br />
             <div class="total-cost">
                 <small>
-                <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)">
-                <h4 id="shares-price"></h4></span>
+                    <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)">
+                    <h4 id="shares-price"></h4></span>
```

Comment:
Since the `<span>` tag surrounds the `<h4>` tag, I would additionally format this to look like:
```html
<span data-tooltip ...>
    <h4 id="shares-price"></h4>
</span>
```

So basically adding `</span>` on its own line and indenting in `<h4>`.